main.go

```diff
@@ -30,6 +31,9 @@ func main() {
 }
 
 func main1() int {
+	gen.Flag("print-commands", "print the commands").Short('x').BoolVar(&log.PrintCommands)
+	gen.Flag("verbose", "print the names of packages as they are generated").Short('v').BoolVar(&log.Verbose)
```

Comment:
Not sure if these should be here or in an init() function?

Reply:
I don't think there's a big difference for a main package. Here seems fine.